libexec/revng/mass-testing/test-harness

```diff
@@ -220,7 +220,9 @@ def main():
     set_child_subreaper()
 
     final_cmd = [*time_cmd, *[p if p != "%INPUT%" else args.input for p in cmd]]
-    if args.memory_limit is None:
+    if "TEST_HARNESS_MEMORY_LIMIT" in os.environ:
```

Comment:
These need to be documented.
I think it might be preferable to have `REVNG_MASS_TESTING_OPTIONS`, simlarly to `REVNG_OPTIONS`. This will save us to end up creating a new environment variable for each option, which is kinda frustrating.

Reply:
Ok on using the `OPTIONS` env, is it ok if it's named `REVNG_TEST_HARNESS_OPTIONS`? I think it'd be clearer

Reply:
Sure

Reply:
Implemented